modules/oe_translation_epoetry/modules/oe_translation_epoetry_mock/oe_translation_epoetry_mock.info.yml

```diff
@@ -0,0 +1,9 @@
+name: OpenEuropa Translation ePoetry Mock
+description: EPoetry mock.
+package: OpenEuropa
+
+type: module
+core_version_requirement: ^9.2
+dependencies:
+  - oe_translation_epoetry:oe_translation_epoetry
+  - oe_translation_remote_test:oe_translation_remote_test
```

Comment:
Try to not depend from oe_translation_remote_test . From what I tried most tests pass already.

Reply:
`EpoetryTranslationMockHelper` extends the `TestRemoteTranslationMockHelper` for the moment. It might be uncoupled in the notifications ticket later on when we stop using this helper in favor of actual soap notification requests. For now we can keep the dep.